src/app/pages/wallets/my-wallets/pages-wallets-my-wallets.component.ts

```diff
@@ -44,31 +44,35 @@ export class PagesWalletsMyWalletsComponent implements OnInit {
     });
   }
 
-  public addNewWallet({ name }: IWalletModalData): void {
-    this.myWalletsService.addNewWallet(MyWallet.create({ name })).subscribe( (wallet: MyWallet) => {
+  public createWallet({ name }: IWalletModalData): void {
+    this.myWalletsService.createWallet(MyWallet.create({ name })).subscribe( (wallet: MyWallet) => {
         this.myWalletsData.data = [wallet, ...this.myWalletsData.data!]
+    }, () => {
+        console.log('Info for user xd');
     });
   }
 
   public updateWallet({ id }: MyWallet, { name }: IWalletModalData): void {
-    this.myWalletsService.updateNewWallet(MyWallet.create({ id: id!, name })).subscribe( (updatedWallet: MyWallet) => {
+    this.myWalletsService.updateWallet(MyWallet.create({ id: id!, name })).subscribe( (updatedWallet: MyWallet) => {
         this.myWalletsData.data = this.myWalletsData.data!.map(walletItem => {
             if (walletItem.id === id) {
                 return updatedWallet;
             }
 
             return walletItem;
         });
+    }, () => {
```

Comment:
You can remove those error handlers for now. We will introduce them later when some pop ups are introduced. Also using it that way is deprecated. Please check in `ngOnInit` how `next` and `error` should be handled :)